.github/workflows/maven-smoke-test.yml

```diff
@@ -1,61 +1,28 @@
-                  # inplace replacement of the snippet version (latest release) with current version (snapshot)
-                  xmlstarlet edit --inplace --update "//extensions/extension[artifactId='common-custom-user-data-maven-extension']/version" --value $CURRENT_VERSION  common-gradle-enterprise-maven-configuration/.mvn/extensions.xml
-            - name: Run Maven build using CCUD extension
-              run: |
-                  echo 'MAVEN_OUTPUT<<EOF' >> $GITHUB_ENV
-                  mvn -f common-gradle-enterprise-maven-configuration/pom.xml -X --batch-mode clean validate -Dgradle.scan.disabled=true >> $GITHUB_ENV
-                  echo 'EOF' >> $GITHUB_ENV
-            - name: Validate extension loaded
-              run: |
-                  echo $MAVEN_OUTPUT | grep "$LOG_CCUD_EXTENSION_LOADED"
-            - name: Validate extension executed
-              run: |
-                  echo $MAVEN_OUTPUT | grep -L "$LOG_CCUD_EXECUTION_FAILURE"
 
     verify-maven-samples:
         runs-on: ubuntu-latest
         env:
-            LOG_CCUD_EXTENSION_LOADED: "Build operation 'Gradle Enterprise mojo execution' completed"
-            LOG_CCUD_EXECUTION_FAILURE: "Error executing a GradleEnterpriseListener callback"
-            # set GITHUB_TOKEN to avoid maven security error
-            GITHUB_TOKEN: "foo"
+            LOG_MSG_EXTENSION_LOADED: "Build operation 'Gradle Enterprise mojo execution' completed"
+            LOG_MSG_EXECUTION_FAILED: "Error executing a GradleEnterpriseListener callback"
 
         strategy:
             matrix:
                 include:
-                    - relative-path-to-sample-build-file: "default"
-                    - relative-path-to-sample-build-file: "capture-os-processes/maven-os-processes.groovy"
-                    - relative-path-to-sample-build-file: "capture-quality-check-issues/maven-quality-check-issues.groovy"
-                    - relative-path-to-sample-build-file: "capture-top-level-project/maven-top-level-project.groovy"
+                    - project-basedir: common-gradle-enterprise-maven-configuration
+                      relative-path-to-sample-build-file: "default"
+                    - project-basedir: common-custom-user-data-maven-extension
+                      relative-path-to-sample-build-file: "default"
+                    - project-basedir: common-custom-user-data-maven-extension
+                      relative-path-to-sample-build-file: "capture-os-processes/maven-os-processes.groovy"
+                    - project-basedir: common-custom-user-data-maven-extension
+                      relative-path-to-sample-build-file: "capture-quality-check-issues/maven-quality-check-issues.groovy"
+                    - project-basedir: common-custom-user-data-maven-extension
+                      relative-path-to-sample-build-file: "capture-top-level-project/maven-top-level-project.groovy"
```

Comment:
maven profiles is missing

Reply:
It was merged before maven profiles sample was introduced